assets/js/googlesitekit/site/index.js

```diff
@@ -0,0 +1,210 @@
+/**
+ * External dependencies
+ */
+import invariant from 'invariant';
+
+/**
+ * Internal dependencies
+ */
+import { get, set } from 'googlesitekit-api';
+import {
+	FETCH_CONNECTION_INFO,
+	FETCH_RESET,
+	INITIALIZE,
+	RESET,
+	RESET_FAILURE,
+	RESET_SUCCESS,
+	RECEIVE_CONNECTION_INFO,
+	RECEIVE_CONNECTION_INFO_FAILED,
+} from './index.private';
+
+export const INITIAL_STATE = {
+	connectionInfo: null,
+	isFetchingConnectionInfo: false,
+	isResetting: false,
+};
+
+export const STORE_NAME = 'core/site';
+
+export const actions = {
+	fetchConnectionInfo() {
+		return {
+			payload: {},
+			type: FETCH_CONNECTION_INFO,
+		};
+	},
+
+	fetchReset() {
+		return {
+			payload: {},
+			type: FETCH_RESET,
+		};
+	},
```

Comment:
+1 on this naming of `fetch...` and `receive...` for issuing API requests and receiving responses. Let's stick with that throughout.